wp-graphql-smart-cache.php

```diff
@@ -209,7 +210,7 @@ function () {
 add_action(
 	'graphql_purge',
 	function ( $purge_keys ) {
-		if ( ! function_exists( 'graphql_get_endpoint_url' ) || ! class_exists( 'WpeCommon' ) || ! method_exists( 'WpeCommon', 'http_to_varnish' ) ) {
+		if ( ! function_exists( 'graphql_get_endpoint_url' ) || ! class_exists( 'WpeCommon', false ) || ! method_exists( \WpeCommon::class, 'http_to_varnish' ) ) {
```

Comment:
This change to ::class probably not necessary once I figured out needed the stub file for wpe common.  But shouldn't be a change in behavior.  I can back out if you think I should.

Reply:
@markkelnar I think it's fine.